android/fastlane/Fastfile

```
@@ -1,5 +1,22 @@
+    aabPath = "../../build/app/outputs/bundle/debug/app-debug.aab"
+    FileUtils.remove_dir(aabPath) if File.directory?(aabPath)
+
+    sh("flutter", "build", "appbundle", "--debug")
+    unless File.exist?(aabPath)
+      UI.user_error!("Build failed")
+    end
+  end
+
+  desc "Deploy a new version to the Google Play"
+  lane :internal do
+    version_codes = google_play_track_version_codes(track: "internal")default_platform(:android)
```

RuboCop (1.81.7)

[fatal] unexpected token tIDENTIFIER
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)